docs/project-development-environment.md

```diff
@@ -2,7 +2,19 @@
 
 ## Local prerequisites
 
-Docker and docker-compose is required to be installed.
+| dependency | supported versions | Installation recommendation for Ubuntu 14.04 |
+| ---------- | ------------------ | ---------------------------------------------|
+| <a name='dependency_docker'>docker</a> | 1.10 (1.9 may still work, too) | Follow [the official Docker installation instructions for Linux](https://docs.docker.com/linux/step_one/) |
+| <a name='dependency_docker-compose'>docker-compose</a> | 1.6 or better | Install system-wide via <a href='#dependency_pip'>`pip`</a> (**not** via `pip3`!): <pre class="highlight highlight-source-shell">sudo pip install docker-compose</pre> |
```

Comment:
> "1.6 or newer" (let's hope the new versions are better ;-)

Actually, I consciously chose "or better" exactly because newer versions in the past have not always been better (for us / for our then-current docker-compose.yml files) due to (sometimes subtle) backwards incompatibilities. Stuffing this semantic into the phrasing is rather obscure, though, so I'll change this to just "1.6".

As the column is labeled "supported versions", not "required versions", that will capture the intended meaning sufficiently well. We can start "supporting" other versions of docker-compose once they're out and have been tested by us.